### PR TITLE
fix logout login without refreshing page throwing error

### PR DESCRIPTION
**Pull request by codeliger**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -49,25 +46,34 @@
...
+            })
             let callback = (frames) => {
-                let close = slider.Load(frames)
+                if (frames instanceof Array && frames.length > 0){
+                    let close = slider.Load(frames)
+                }
             }
-            tfr.TryOn("5003-008-14", callback)
+            document.getElementById("tryon").addEventListener("click", (e) => {
+                tfr.TryOn("5003-007-08", callback)
```
Comment (edalford11) [inline index.html:69]: Is this supposed to be here?

Comment (codeliger) [reply index.html:69]: everything in index.html is just for local testing so its fine there

**Review by matheuslino — APPROVED** [review]
All the comments I've made don't affect the PR. The majority of them are questions and suggestions for future modification.

File: index.html [hunk 2]
```diff
@@ -32,15 +30,14 @@
...
         }
         window.onSignIn = (colorwaySizeAssetSKU) => (username, password, validationError) => {
-            if(username.length == 0 || password.length == 0) {
+            if (username.length == 0 || password.length == 0) {
                 validationError("Username or password is empty")
             }
-            console.log("sign in", colorwaySizeAssetSKU, username, password)
         };
         window.onNavSignIn = (colorwaySizeAssetSKU) => (email) => {
             console.log("nav sign in", colorwaySizeAssetSKU, email)
```
[the remark this answers is not in the document]
Comment (codeliger) [reply index.html:38]: we didnt make a decision on this yet.

we can also change all console.log to console.debug and most compilers will strip it during production builds

File: index.html [hunk 3]
```diff
@@ -49,25 +46,34 @@
...
+            let signout = document.getElementById("signout")
+
             let onChange = (slider, imageUrl) => {
                 console.log("slider change", slider, imageUrl)
                 tryOnImage.src = imageUrl
             }
             let slider = comps.InitImageSlider("tfr-slider", onChange)
-            let tfr = InitFittingRoom(1, "thefittingroom-modal")
+            let tfr = InitFittingRoom(1, "tfr-tryon-modal")
+            signout.addEventListener("click", (e) => {
```
[the remark this answers is not in the document]
Comment (codeliger) [reply index.html]: removed

File: index.html [hunk 4]
```diff
@@ -49,25 +46,34 @@
...
+                tfr.user.SignOut()
+            })
             let callback = (frames) => {
-                let close = slider.Load(frames)
+                if (frames instanceof Array && frames.length > 0){
```
[the remark this answers is not in the document]
Comment (codeliger) [reply index.html]: changed

**Pull request merged**